lib/wallaroo/core/checkpoint/checkpoint_initiator_phase.pony

```diff
@@ -232,6 +242,14 @@ class _RollbackCheckpointInitiatorPhase is _CheckpointInitiatorPhase
   =>
     None
 
+  fun ref start_checkpoint_timer(time_until_next_checkpoint: U64,
```

Comment:
I address this at 5015c02106502a8beb9beeaee577c8bdde6308fc where I delay the timer, but it's probably better that we ignore the timer rather than delay it, provided we're guaranteeing that we start a new timer when rollback is complete.  Let's discuss this.

Reply:
What I mean to say is that I think your change should take precedence over mine.